include/models.h

```diff
@@ -36,9 +36,17 @@ class Model {
 
     void config(SearcherConfig &config_, const std::vector<std::vector<int>> &stopWordsList_ = {});
 
+    void set_input(std::vector<int32_t> &inputIds_, int batchSize_, int maxLen_ = -1, int numBeams_ = 1,
+            int numBeamHypsToKeep_ = 1, float lenPenalty_ = 1.0, bool doEarlyStopping_ = false, int eosTokenId_ = -1,
+            int padTokenId_ = -1, bool doSample_ = false, float temperature_ = 1.0, int topK_ = 50, float topP_ = 1.0,
+            float repetitionPenalty_ = 1.0, const std::vector<std::vector<int>> &stopWordsList_ = {});
+
+    void set_input(std::vector<int32_t> &inputIds_, int batchSize_, SearcherConfig &config_,
+            const std::vector<std::vector<int>> &stopWordsList_ = {});
+
     bool isDone();
 
-    std::tuple<float *, int, int> forward();
+    std::tuple<float *, int, int> forward(bool logits_all = true);
```

Comment:
default to false

Reply:
Default to true for high level API evaluation.

Reply:
Keep align with existed path, will check and change in the future.